rs/cli/src/runner.rs

```diff
@@ -444,4 +445,30 @@ impl Runner {
             .await?;
         Ok(())
     }
+
+    pub async fn network_heal(
+        &self,
+        request: ic_management_types::requests::HealRequest,
+        _verbose: bool,
+        simulate: bool,
+    ) -> Result<(), anyhow::Error> {
+        let change = self.dashboard_backend_client.network_heal(request).await?;
+        println!("{}", change);
+
+        join_all(change.subnets_change_response.iter().map(|subnet_change_response| async move {
```

Comment:
So we're collecting the replacement results in parallel? Is this time consuming, or why do we do it in parallel? I don't mind, I'm just curious.

Reply:
Well actually the only reason why I am doing the join_all is to execute the run_membership_change inside the map...it is actually sequential execution of the proposals.